Mayden-Academy/2020-jan-makeAMealOfIt
# S2 t1 front end

Front end ready for code review.

To test:
Display front end HTML in various browsers.
Resize to below 800px to check media query results in divs expanding to 100% width.
Test link to recipepuppy.com
When clicking "SEARCH", you should be taken to the non-existent page "recipes.php"
